### PR TITLE
179: add-model-to-sites-table

### PR DESCRIPTION
**Pull request by hanaawad24**
# Pull Request

## Description

This PR addresses the issue #179 to handle the ability to assign a model to one or more site.

### CODE REVIEW

**Comment by peterdudfield** [comment]
Thanks so much @hanaawad24 for doing this. 

I've added this in already - https://github.com/openclimatefix/pv-site-datamodel/blob/main/pvsite_datamodel/write/user_and_site.py#L381

I wondered if you could improve the current function, and include those extra tests you have added?

File: pvsite_datamodel/sqlmodels.py [hunk 1]
```diff
@@ -182,13 +181,7 @@ class SiteSQL(Base, CreatedMixin):
...
         comment="The UUID of the client this site belongs to",
     )
-
-    ml_model_uuid = sa.Column(
-        UUID(as_uuid=True),
-        sa.ForeignKey("ml_model.model_uuid"),
-        nullable=True,
-        comment="The ML Model which should be used for this site",
-    )
+    model_uuid = sa.Column(UUID(as_uuid=True), sa.ForeignKey("ml_model.model_uuid"))
```
Comment (peterdudfield) [inline pvsite_datamodel/sqlmodels.py:184]: could you remove this change, the nullable=True, and the comment is useful

**Comment by peterdudfield** [comment]
I'll close this for the moment, but feel free to add those test improvements on this PR, or a new one

**Comment by hanaawad24** [comment]
> I'll close this for the moment, but feel free to add those test improvements on this PR, or a new one

@peterdudfield Thank you for your time and feedback. I did another pull request #186  Thank you

**Pull request closed without merging**